.github/workflows/ci.yml

```diff
@@ -22,17 +22,17 @@ jobs:
       #          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
       #          GITHUB_TOKEN: ${{ github.token }}
       #        run: ./gradlew sonarqube -Dsonar.projectKey=xenit-eu_alfresco-gradle-sdk -Dsonar.organization=xenit-eu -Dsonar.host.url=https://sonarcloud.io
-#      - name: Publish
-#        if: ${{ github.ref == 'refs/heads/master' || startswith(github.ref, 'refs/tags/') }}
-#        env:
-#          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.MAVEN_CENTRAL_GPG_KEY }}
-#          ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.MAVEN_CENTRAL_GPG_PASSWORD }}
-#          ORG_GRADLE_PROJECT_sonatype_username: ${{ secrets.MAVEN_CENTRAL_USERNAME }}
-#          ORG_GRADLE_PROJECT_sonatype_password: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
-#        run: ./gradlew publish
-#      - name: Publish docker images
-#        if: ${{ github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/') }}
-#        env:
-#          DOCKER_USER: ${{ secrets.DOCKER_USER }}
-#          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-#        run: ./gradlew pushDockerImage
\ No newline at end of file
+      - name: Publish
+        if: ${{ github.ref == 'refs/heads/master' || startswith(github.ref, 'refs/tags/') }}
+        env:
+          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.MAVEN_CENTRAL_GPG_KEY }}
+          ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.MAVEN_CENTRAL_GPG_PASSWORD }}
+          ORG_GRADLE_PROJECT_sonatype_username: ${{ secrets.MAVEN_CENTRAL_USERNAME }}
+          ORG_GRADLE_PROJECT_sonatype_password: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
+        run: ./gradlew publish
+      - name: Publish docker images
```

Comment:
What docker images are we publishing?

If it's just alfresco/share images with the telemetry AMP installed, please don't publish them.
*We* won't use those images, but just install the necessary AMPs in our projects, as we already have to install other custom AMPs into Alfresco.
I think it's the responsibility of the user of those images to install the AMPs, because pretty much all users will have more than just the telemetry AMP installed into Alfresco.

Do we really need an additional set of Alfresco images to keep up-to-date and for which we will publish all supported versions?

Reply:
It's just alfresco/share images with the zipkin AMP, yes.
I agree with that view.

My first intention was to just translate the Jenkinsfile into a GH ci.yml.
But now is indeed a good opportunity to make a decision on whether we should be publishing those images.

Reply:
adding a commit already, other reviewers can complain about it if they want